README.md

```diff
@@ -13,6 +14,28 @@
 - [Troubleshooting](#troubleshooting)
   * [All Slaves are marked as offline](#all-slaves-are-marked-as-offline)
 
+
+## Create a persistent disk Jenkins home ##
+
+Change volumes partition from 1 to 0 in the file k8s/jenkins/jenkins.yaml\<br> 
+
+Run script/create_backup_pd with the following parameters 
+
+Usage: 
```

Comment:
Please fix mardown such that output of script/create_backup_pd -h is grayed out.